src/database/hooks/useLocalDatabaseHook.js

```diff
@@ -10,14 +10,15 @@ import {InitialStartupAtom} from '../../service/initialStartup';
 
 const MIGRATION_STATUS = {
   NOT_MIGRATED: 'NOT_MIGRATED',
-  MIGRATED: 'MIGRATED'
+  MIGRATED: 'MIGRATED',
+  MIGRATING: 'MIGRATING'
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a new status 'MIGRATING' to the MIGRATION_STATUS enum is a good practice. It allows for better tracking of the migration process and can prevent potential issues such as multiple migrations happening at the same time.

```diff
+  MIGRATING: 'MIGRATING'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->